app/views/users/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: user, local: true) do |form| %>
+  <% if user.errors.any? %>
```

Comment:
please remove all the logic from your views and put it inside the view helpers `app/helpers/*_helper.rb`

Reply:
Done